Cambios a el dept de ciencias +arrelga el maint de ciencias + arregla la configuracion de las torretas del satelite + actualiza el minimapa.

Evankhell561 opened the pull request:
:cl:
tweak: robotica es mas grande.
add: se añadio una sala de reuniones en ciencias.
add: se agregaron 2 espacios para mechas en mechbay
add: se agrego un tanque de aceite en mech bay.
add: se agrego un tanque de agua a mechbay.
add: se agrego un calentador a toxinas.
add: se agrego un penkiller a xenobiologia.
del: se eliminaron los beakers de acido de los circuit imprimer.
add: se agrego una caja de herramientas, una multitool y un cinturon la oficina del RD.
add: añade cascaras de granadas a quimica.
add: se añade cables y señalizadores en quimica
add: se añade lentes de ciencia al experimentor
tweak: ahora los cientificos tienen acceso a su propio mantenimiento.
tweak: las torretas del satelite de comunicaciones ahora no atacaran a todos los no sinteticos.
tweak: añade los cambios al minimapa
fix: el Rd tiene acceso al boton de su puerta,
tweak: todos los heas tienen acceso al eva.
/:cl:

Este es un cambio al nuestro mapa donde aprovecha mas el espacio que hay en ciencias para ampliar mechbay y robotica. tambien agregue cosas a los otros laboratorios que hacian falta para 
al final hice mas cambios a toda ciencias, puede verse en las imagenes. 

entre estos cambios los maas importantes son que arregle un error de mapeo en toxians e hice una penkiller en xenobiologia:
.
![granrobotica](https://user-images.githubusercontent.com/49106866/56477288-ee15d100-6471-11e9-83cc-b5ad6e2c6be6.png)
![granquimica](https://user-images.githubusercontent.com/49106866/55799760-272c6980-5aa0-11e9-8b94-768e53600ac8.png)
![quimica](https://user-images.githubusercontent.com/49106866/55696327-6d8fa480-598a-11e9-885a-34a0612e1243.png)
![RD](https://user-images.githubusercontent.com/49106866/56477292-fb32c000-6471-11e9-802f-e4e978480f51.png)
![grantoxinas](https://user-images.githubusercontent.com/49106866/55799732-17148a00-5aa0-11e9-80cb-a2128cdf8451.png)
![toxinas](https://user-images.githubusercontent.com/49106866/55696330-6d8fa480-598a-11e9-854f-e15b7a77cb3b.png)
![granxenobiologia](https://user-images.githubusercontent.com/49106866/55875612-79828e80-5b63-11e9-8caa-4818d06f5575.png)
![xenobiologia](https://user-images.githubusercontent.com/49106866/55696332-6e283b00-598a-11e9-8fe7-de7a3c2f75d7.png)
![pen slime](https://user-images.githubusercontent.com/49106866/55798676-89d03600-5a9d-11e9-9d70-50754fb5cd39.png)
![granexperimentor](https://user-images.githubusercontent.com/49106866/55800040-c2254380-5aa0-11e9-8299-5b0e7fa27bbc.png)
![experimentor](https://user-images.githubusercontent.com/49106866/55741820-2c37dd00-59fc-11e9-913d-8f4c3c49f669.png)
![saladeestar](https://user-images.githubusercontent.com/49106866/56732969-d1f18880-672c-11e9-9367-57d0cf28657f.png)
es.githubusercontent.com/49106866/55875637-84d5ba00-5b63-11e9-833b-6579872d65ee.png)
![serverroom](https://user-images.githubusercontent.com/49106866/56477302-1998bb80-6472-11e9-9add-4ddd817fed2b.png)
![depositotoxinas](https://user-images.githubusercontent.com/49106866/56732982-d74ed300-672c-11e9-9830-34321abc4e98.png)
![minimap](https://user-images.githubusercontent.com/49106866/56735520-afaf3900-6733-11e9-81a2-04565906444e.png)

Commit:
Update hispania.dmm

Commit:
modifica robotica

se romovieron los cargadores de robotica.
actualmente hay espacio para un total de 4 mechas.
no se le quita espacio a genetica.
se agrego un tanque de acietie
se cambio un muro por una ventana en robotica.

Commit:
newrobotica1

Commit:
quita el acido de los circuit imprimer

Evankhell561 commented:
tampoco entiendo por que el fallo ahora. a mi no me da errores.

Tilk1 commented:
Debo decir que a nivel diseño se ve increible y queda muy bonito.
Pero analizando los pros y contras no le veo sentido a tal cambio.
Pros: hay 2 cargadores de mechas mas.  Me gusta que E-N casi es deleteado
Cons: Esos cargadores muy rara ves se usan y pueden ser construidos. Le roba espacio a otro departamento. (Aunq sean 2 tiles, es bastante). Pierde simetria geneticas. Roboticas ahora se ve enorme y los cyborgs deben caminar mucho mas para ir a recargarse

Commit:
actualiza muchas partes de ciencias.

añade heater a toxinas. cascaras de granada a quimica. un pen kill a xeno. y regresa una tile a genetica.

Evankhell561 commented:
ya no le robo nada a genetica. todo esta dentro del espacio de ciencias y agrege cosas que hacian falta en el departamento para su correcto funcionamiento.

Commit:
ultimos ajustes

añade un timer que faltaba a quimica
fixea el pen kill.

Commit:
arregla la  los cambios a la oficina del rd

Commit:
agrega una multitoll a la oficina del rd

Commit:
hace a roboticca mas amplia

Commit:
añade calbes y mueve la silla de robotica

Commit:
arregla un defecto del mapeo.

Commit:
arregla un bug de toxinas.

iRyzor commented:
Aparte de poner imágenes donde se hicieron los cambios, podrías poner imágenes de un gran área para tener mas visión de los cambios y verlo junto al resto de cosas que no han cambiado.

Commit:
Merge branch 'master' into robotica

Commit:
Update hispania.dmm

Commit:
Update hispania.dmm

Commit:
Update hispania.dmm

Commit:
Update hispania.dmm

Commit:
Update hispania.dmm

Commit:
agrega sala de estar, repara step.

Commit:
cablea la sala de star

Commit:
Update hispania.dmm

Commit:
Update hispania.dmm

Commit:
Update hispania.dmm

Commit:
Merge branch 'master' into robotica

Commit:
restablece el master

Commit:
Merge branch 'master' into robotica

Commit:
Update hispania.dmm

Commit:
Update hispania.dmm

Commit:
Update hispania.dmm

Commit:
Merge branch 'master' into robotica

Commit:
Merge branch 'master' into robotica

Commit:
Update hispania.dmm

Commit:
Update hispania.dmm

Commit:
toques finales

Commit:
pone las areas de la sala de estar

Commit:
Update hispania.dmm

Commit:
Update hispania.dmm

Commit:
actualiza la sala de estar.

pone una puerta de incendios en la entrada.
elimina una maquina y mueve las demas.

Commit:
Update hispania.dmm

Dapple27 commented:
no sabia que el maint de ciencias estaba dañado :thinking:

Commit:
Update z3.dmm

Commit:
Update z3.dmm

Commit:
arregla las areas de la expansion de robotica

Commit:
Update hispania.dmm

Commit:
Update hispania.dmm

Commit:
Update hispania.dmm

Commit:
arregla las areas de la sala de estar

arregla las areas de la sala de estar para que sean unicas. añade un floor en la entrada de dicha sala, en coomit anteriores se expandio el rango de la pc de toxinas a 30 y se movio al sur del seposito de toxinas

Commit:
lo mismo que el anteriore por con mapremate

Commit:
Merge branch 'master' into robotica

Commit:
Update Hispania_nanomap_z1.png

Commit:
agrega el minimapa hecho por programa interno

Commit:
Merge branch 'master' into robotica

Commit:
Update hispania.dmm
